project-base/src/Controller/Front/CartController.php

```diff
@@ -406,4 +417,20 @@ private function isCartHoverEnable(): bool
 
         return !in_array($masterRequest->get('_route'), self::PAGES_WITH_DISABLED_CART_HOVER, true);
     }
+
+    /**
+     * @required
+     * @param \Shopsys\FrameworkBundle\Model\Module\ModuleFacade $moduleFacade
+     * @internal This function will be replaced by constructor injection in next major
```

Comment:
```suggestion
     * @deprecated This function will be replaced by constructor injection in next major
```

Reply:
It should be internal as it is called by DIC.